Components/Utils/ModelRenderer.cs

```diff
@@ -125,13 +126,36 @@ public HtmlString UnilingualPreFooter()
 
         public HtmlString PreFooter()
         {
+            Feedback feedback = new Feedback();
+            feedback.Enabled = _model.Settings.FeedbackLink.Show;
+
+            if (!string.IsNullOrWhiteSpace(_model.Settings.FeedbackLink.Text) || !string.IsNullOrWhiteSpace(_model.Settings.FeedbackLink.Section) || !string.IsNullOrWhiteSpace(_model.Settings.FeedbackLink.Theme))
+            {
+                feedback.Section = _model.Settings.FeedbackLink.Section;
+                feedback.Theme = _model.Settings.FeedbackLink.Theme;
+                if (Thread.CurrentThread.CurrentUICulture.TwoLetterISOLanguageName.StartsWith(Constants.FRENCH_ACCRONYM, StringComparison.OrdinalIgnoreCase))
```

Comment:
Text and Url should have same logic as href (ie depends on culture, but if no fr is specified the english one should be used)

Reply:
done!

Reply:
Sorry to be a pest, this is actually fine, but since we're repeating the Thread thing multiple times and it is quite long, we could create `var isFrenchCulture = Thread.CurrentThread.CurrentUICulture.TwoLetterISOLanguageName.StartsWith(Constants.FRENCH_ACCRONYM, StringComparison.OrdinalIgnoreCase)` and use `isFrenchCulture` where needed instead.   

Thoughts?

Reply:
Yup, its a good change. I'll update it

Reply:
done!